### PR TITLE
Updated lev syops for change of monarch title

### PR DESCRIPTION
**Pull request by aarongill1**

### CODE REVIEW

**Comment by sureshnature** [comment]
The email address you have sent to ***@***.*** is for a legacy domain and will shortly be removed.

Please update your address book and send to the same address but having removed .gsi. from the domain part.

Mail to the legacy addresses will stop working in a few weeks.

**Comment by conwubiko** [comment]
The email address you have sent to ***@***.*** is for a legacy domain and will shortly be removed.

Please update your address book and send to the same address but having removed .gsi. from the domain part.

Mail to the legacy addresses will stop working in a few weeks.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "keycloak-theme-govuk",
-  "version": "2.0.9",
+  "version": "2.0.20",
```
Comment (ashley-bickerstaff-homeoffice) [inline package.json:3]: Although the repo homepage says 2.0.19 is the latest, on the releases page, it says that the *.20 tag already exists. Perhaps this should be *.21?

**Review by ashley-bickerstaff-homeoffice — CHANGES_REQUESTED** [review]
Can you check that the versioning won't be an issue?

**Review by ashley-bickerstaff-homeoffice — APPROVED** [review]

File: package.json
Comment (James-Royds-HO) [inline package.json:1]: This version bump isn't needed. Just your changes to your T&Cs

**Review by James-Royds-HO — APPROVED** [review]

**Pull request merged**